packages/block-library/src/button/index.js

```diff
@@ -78,7 +78,7 @@ export const settings = {
 	},
 
 	styles: [
-		{ name: 'default', label: _x( 'Rounded', 'block style' ), isDefault: true },
+		{ name: 'default', label: _x( 'Default', 'block style' ), isDefault: true },
```

Comment:
I think we landed a PR recently where we could drop these entirely from here and they will be added automatically (needs verification)

Reply:
Can we get a reference to that PR here? What exactly can we drop?

Reply:
Sure sorry it wasn't that clear.

Here's the PR https://github.com/WordPress/gutenberg/pull/12519

I think the entire line can be dropped (the default style)

Reply:
I just tested this, and it _almost_ works. The block has the correct styles by default, but the "Default" style option does not appear in the style picker: 

![screen shot 2019-02-05 at 12 51 47 pm](https://user-images.githubusercontent.com/1202812/52293519-0b440480-2945-11e9-9fec-b22a3df79c56.png)

Since it's not there, if you select an alternate style, there'd be no way to switch back to the default. 😕 

So for now at least, I think we do need these lines.

Reply:
Mmm weird, I though it was the intent of the linked PR. pinging @swissspidy to check my assumptions :) but feel free to move forward with the PR.

Reply:
Yeah that was the goal of #12519 🤔 No idea why it doesn't work, I would have to investigate.